src/GitVersionExe/Program.cs

```diff
@@ -129,17 +130,27 @@ static void ConfigureLogging(Arguments arguments)
             {
                 try
                 {
-                    Directory.CreateDirectory(Path.GetDirectoryName(arguments.LogFilePath));
-                    if (File.Exists(arguments.LogFilePath))
+                    var logFileFullPath = Path.GetFullPath(arguments.LogFilePath);
+                    var logFile = new FileInfo(logFileFullPath);
+
+                    // NOTE: logFile.Directory will be null if the path is i.e. C:\logfile.log. @asbjornu
+                    if (logFile.Directory != null)
+                    {
+                        logFile.Directory.Create();
+                    }
+
+                    if (!logFile.Exists)
```

Comment:
@JakeGinnivan, @orjan Yea, I can remove the `if`. But given [line 155 in `master`](/GitTools/GitVersion/blob/master/src/GitVersionExe/Program.cs#L155):

```c#
File.AppendAllText(arguments.LogFilePath, contents);
```
I find the old behavior of replacing the file weird. Wouldn't this make the code a bit more intuitive?

```c#
File.WriteAllText(arguments.LogFilePath, contents);
```